lib/Omise.php

```diff
@@ -1,4 +1,126 @@
 <?php
+namespace Omise;
```

Comment:
Note, I implement this `Omise\Omise` class at the same place with those class include part because the following reason.
1. As for the name, I think `Omise\Omise` is the right name for this configuration.
2. However, this `lib/Omise.php` file is, normally a file where we suggest all developers to load into their project (as for manual installation) in order to use Omise-PHP.

All inputs are welcomed, if you have any idea for a better place or name for this class.